.github/workflows/e2etests.yaml

```diff
@@ -0,0 +1,127 @@
+name: Meshery Adapter for consul Tests
+
+on:
+  push:
+    branches:
+      - "*"
+    tags:
+      - "v*"
+  pull_request:
+    branches:
+      - "*"
+  release:
+    types: [published]
+jobs:
+  SetPatternfileForServicemeshInstall:
+    runs-on: ubuntu-latest
+    outputs:
+      sm_version: ${{ steps.gettag.outputs.release }}
+      adapter_version:  ${{ env.version }}
+    steps:
+      - name: Checkout Code
+        uses: actions/checkout@v2
+      - name: Get version of adapter
+        run: |
+          if [ ${{ github.event_name }} == "release" ];then
+              echo "version=${GITHUB_REF/refs\/tags\//}" >> $GITHUB_ENV
+          else 
+              echo "version=edge" >> $GITHUB_ENV
+          fi    
+      - name: Get latest release tag 
+        id: gettag  
+        uses: pozetroninc/github-action-get-latest-release@master
+        with:
+          repository: hashicorp/consul
+          excludes: prerelease, draft, edge  
+      - name: Change service mesh version in patternfile
+        run: | 
+           yq e -i '.services.consul.settings.version="${{ steps.gettag.outputs.release }}"' ./.github/install/deploy.yaml 
+           cat ./.github/install/deploy.yaml  
+      - name: Uploading file
+        uses: actions/upload-artifact@v2
+        with:
+          name: patternfile
+          path: ./.github/install/deploy.yaml
+
+  TestAdapterForServicemeshInstall:
+    needs: SetPatternfileForServicemeshInstall
+    uses: meshery/meshery/.github/workflows/testadapters.yaml@master  
+    with: 
+      expected_pods: consul-connect-injector-webhook-deployment, consul-controller, consul-server, consul-webhook-cert-manager, consul, prometheus-server
+      expected_pods_namespaces: consul, consul, consul, consul, consul, consul
+      deployment_url: https://github.com/meshery/meshery/blob/master/install/deployment_yamls/k8s/meshery-consul-deployment.yaml
+      service_url: https://github.com/meshery/meshery/blob/master/install/deployment_yamls/k8s/meshery-consul-service.yaml
+      adapter_name: consul
+      patternfile_name: deploy.yaml 
```

Comment:
Please also pass `k8s_version` here

Reply:
We don't need to pass k8s_version anymore